cogs/admin.py

```
@@ -0,0 +1,205 @@
+                title="❌ Evaluation Error",
+                description=f"```python\n{str(e)}\n```",
+                color=self.bot.config.ERROR_COLOR
+            )
+            await ctx.send(embed=embed)
+    
+    @commands.command(
+        name="sql",
+        description="Execute SQL query on the database",
+        usage="sql <query>"
+    )
+    @is_owner()
```

Pylint (3.3.7)

[error] Too many positional arguments for method call

(E1121)